config/master.d/master.conf

```diff
@@ -1,5 +1,4 @@
 user: root
-auto_accept: True
```

Comment:
Would it make sense to explicitly set it to `False`?, just in case for whatever reason this default changes in the future